sponsorship_switzerland/models/completion_rules.py

```diff
@@ -350,7 +350,7 @@ def _generate_invoice(self, stmts_vals, st_line, partner, ref_index):
 
     def _generate_invoice_line(self, invoice_id, product, st_line, partner_id):
         inv_line_data = {
-            "name": product.name,
+            "name": st_line["note"] if "note" in st_line and st_line["note"] != "" else product.name,
```

Comment:
One shorter way of doing the same:
```suggestion
            "name": st_line.get("note") or product.name,
```